app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  skip_before_action :verify_authenticity_token, if: proc { :bypass_auth? }
```

Comment:
[issue] I think this is actually always skipping the check. It should probably be either:

`skip_before_action :verify_authenticity_token, if: proc { bypass_auth? }` or

`skip_before_action :verify_authenticity_token, if: :bypass_auth?` or

`skip_forgery_protection if: :bypass_auth?`

Reply:
i don't think that's true, not as it's currently written. (i had double checked in my local env to ensure it was kidding the conditional, and taking those suggestions leads to an error `undefined method `bypass_auth?' for #<PagesController:0x000000000066a8>`) however, i think that *will* work once i move the helper into the ApplicationController as you suggested below.

Reply:
`current_user` and `user_signed_in?` are used in the view, so i am going to leave those in the helper

Reply:
oh actually, since i use bypass_auth in the `current_user` method i don't think i can hm

Reply:
oh wait duh you answered that below, ignore the last comment.